docs/remote-feature-flags.md

```diff
@@ -0,0 +1,171 @@
+# Remote Feature Flags
+
+## Overview
+
+This document outlines the process for adding and managing remote feature flags in MetaMask (Extension and Mobile). Remote feature flags allow us to control feature availability and behavior in production without requiring a new release. They are created on the Launch Darkly platform and distributed by our internal API [ClientConfigAPI](https://github.com/consensys-vertical-apps/mmwp-client-config-api). Please read the [Remote feature flags ADR](https://github.com/MetaMask/decisions/pull/43) for more details.
```

Comment:
I think "LaunchDarkly" is one word, not two.

Reply:
🕵🏻‍♂️ modified in d097ff0f7dd9b7c5908abf7c2e07982e4c04165d